src/components/SwipeClassifier.js

```diff
@@ -73,6 +89,8 @@ export class SwipeClassifier extends React.Component {
     const renderClassifierOrTutorial = () => {
       const key = this.props.workflow.first_task //always just one task
       const task = this.props.workflow.tasks[key]
+      const answers = reverse(task.answers)  //Yes is listed first in project, but we need No listed first (on left)
```

Comment:
Yeah it says "If you have a Yes/No question, we recommend Yes is the first option listed so that it appears on the right."  This is because the existing projects (Steller Sea Watch and P4:ridges) had listed Yes first, I think because it looks better in PFE like that.  So then the swipe app has to reverse that order.  (I think you can access the Insect Classification csv export, and you can check the resulting answers in that)